src/app/components/footer/footer.component.html

```diff
@@ -56,9 +56,9 @@ <h4>Legal</h4>
     </div>
     <div class="row justify-content-center">
       <div class="col-3 copyright">
-        <p>Find <strong>DeX</strong> on <a href="https://github.com/DigitalExcellence" target="_blank" rel="noopener noreferrer"><strong>GitHub</strong></a></p>
+        <p>Find <strong>DeX</strong> on <a class="github-link" href="https://github.com/DigitalExcellence" target="_blank" rel="noopener noreferrer"><strong>GitHub</strong></a></p>
```

Comment:
With noopener and noreferrer you won't be able to [track visitor statistics in the repository itself](https://github.com/DigitalExcellence/dex-frontend/graphs/traffic), but this might be intended.

Reply:
I think it's just 'noreferrer' that prevents us from tracking right?

Reply:
I have no idea I did not make it, I don't know if it was intended

Reply:
Oh yeah, I confused the two. The noopener is to prevent cross site hijacking.

Reply:
Right, then the 'noreferrer' can be removed. My mistake :)

Reply:
Fixed. Thanks for noticing.

Reply:
Looks like SonarQube does not agree with this change. Older browsers need both noreferrer and noopener apparently. 
I will revert my commit for now.

Reply:
Wouldn't that just be older versions of Internet Explorer, in which our complete layout doesn't even work correctly?

Reply:
Possibly, it's something we definitely should look into in the future and set up custom rules in sonar qube if this is true to remove this additional check.